.gitignore

```diff
@@ -145,3 +145,6 @@ cython_debug/
 
 # MacOS
 .DS_Store
+
+# lahman cache
+baseballdatabank-master
```

Comment:
Whoa whoa whoa

Reply:
Is this not preferred? I'm just preventing an accidental checkin of anyone downloading lahman while working on the library. If we want to leave this to enforcement at the PR level we certainly can. Just trying to add some security.

Reply:
fair enough, but this **really** shouldn't happen